xml/art_sle_ha_pmremote.xml

```diff
@@ -0,0 +1,474 @@
+     <para>Verify connectivity between guests and host by using
+      <command>ping</command> and <command>ssh</command>.</para>
+    </step>
+    <step>
+     <para>Install the <package>pacemaker-remote</package> package and
+     enable it:</para>
+     <screen>&prompt.root;<command>zypper</command> in pacemaker-remote
+&prompt.root;<command>systemctl</command> enable pacemaker_remote</screen>
+    </step>
+    <step>
+     <para>Create the <filename>/etc/pacemaker</filename> directory and change
+      the group:</para>
+     <screen>&prompt.root;<command>mkdir</command> -p --mode=0750 /etc/pacemaker
+&prompt.root;<command>chgrp</command> haclient /etc/pacemaker</screen>
+    </step>
+    <step>
+     <para>Run <command>csync2</command> to copy the authentication key from
+      your physical host to your guest node:</para>
+     <screen>&prompt.root;<command>csync</command> -xv</screen>
+    </step>
+    <step>
+     <para>Verify host connection to guest. This can be done by
+      making a SSH connection from the host. This connection
+      will fail, but <emphasis>how</emphasis> it fails gives you hints
+      about its connectivity.
+     </para>
+     <para>Run <command>ssh</command> on one of the cluster nodes:</para>
+     <screen>&prompt.root;<command>ssh</command> -p 3121 &wsII;</screen>
+     <para>An error connection message should be as follows:</para>
+     <screen>ssh_exhange_identification: read: Connection reset by peer.</screen>
+     <para>If you see either of the following messages, the connection does
+      not work:</para>
+     <screen>ssh: connect to host &wsII; port 3121: No route to host
+ssh: connect to host &wsII; port 3121: Connection refused</screen>
+    </step>
+    <step>
+     <para>Shut down the KVM guest as &pace; will manage the guests.
+     </para>
+    </step>
+   </procedure>
+  </sect2>
+
+  <sect2 xml:id="sec.ha.pmremote.integrate.guestsintocluster">
+   <title>Integrating KVM Guests into Cluster</title>
+   <para>To integrate the KVM guests into the cluster, perform the following
+    steps on the physical host <systemitem class="domainname">&node5;</systemitem>:</para>
+   <procedure>
+    <step>
+     <para>Start &pace;:</para>
+     <screen>&prompt.root;<command>systemctl</command> start pacemaker</screen>
+    </step>
+    <step>
+     <para>Check the status of the command <command>crm status</command>. It
+     should contain a running cluster with nodes that are all accessible.
+     </para>
+    </step>
+    <step>
+     <para>Check if all KVM guests nodes and IP addresses are listed
+     in the file <filename>/etc/hosts</filename>.</para>
+    </step>
+    <step>
+     <para>Dump the XML configuration of the KVM guest(s):
+     </para>
+     <screen>&prompt.root;<command>virsh</command> list --all
+ Id    Name         State
+-----------------------------------
+ -     &wsII;       shut off
+&prompt.root;<command>virsh</command> dump &wsII; > /etc/pacemaker/&wsII;.xml</screen>
+    </step>
```

Comment:
This is used in the next step to create the `ocf:heartbeat:VirtualDomain` primitive (in the `config` parameter).